scripts/actions/setup_macaron.sh

```diff
@@ -4,25 +4,17 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
 set -euo pipefail
 
-# Setup Macaron virtualenv and make available via GitHub Actions environment files.
-# This script writes `MACARON=<path>` to `$GITHUB_ENV` so later steps can invoke the macaron CLI, and appends the venv `bin` directory to `$GITHUB_PATH`.
-
 MACARON_DIR="${RUNNER_TEMP:-/tmp}/macaron"
-VENV_MACARON="$MACARON_DIR/.venv/bin/macaron"
 
 mkdir -p "$MACARON_DIR"
+cd "$MACARON_DIR"
 
-if [ -x "$VENV_MACARON" ]; then
-  echo "Using macaron from existing venv: $VENV_MACARON"
-  echo "MACARON=$VENV_MACARON" >> "$GITHUB_ENV"
-  echo "$MACARON_DIR/.venv/bin" >> "$GITHUB_PATH"
-  exit 0
+# Get the run_macaron.sh script
+if [ ! -f "run_macaron.sh" ]; then
+  curl -fSLO https://raw.githubusercontent.com/oracle/macaron/release/scripts/release_scripts/run_macaron.sh
+else
+  echo "run_macaron.sh already exists, skipping download."
 fi
 
-cd "$MACARON_DIR"
-git clone https://github.com/oracle/macaron.git .
-make venv
-export PATH="$MACARON_DIR/.venv/bin:$PATH"
-make setup
-echo "MACARON=$VENV_MACARON" >> "$GITHUB_ENV"
-echo "$MACARON_DIR/.venv/bin" >> "$GITHUB_PATH"
+chmod +x run_macaron.sh
+echo "MACARON=$MACARON_DIR/run_macaron.sh" >> "$GITHUB_ENV"
```

Comment:
If you don't set the `MACARON_IMAGE_TAG` environment variable, this script will [always run the latest](https://github.com/oracle/macaron/blob/84493c44d7a3d2ab6e49fcf87c40ae3aaa246bf1/scripts/release_scripts/run_macaron.sh#L48), which is not what we want. We should check what tag the action is called for and this env variable.